pom.xml

```diff
@@ -44,6 +44,7 @@
        possible to measure cross-module test coverage -->
     <!--suppress UnresolvedMavenProperty -->
     <code.coverage.disabled>false</code.coverage.disabled>
+    <sonar.coverage.exclusions></sonar.coverage.exclusions>
```

Comment:
If I understand it correctly, this line made the coverage working, right?

I wonder why it behaves like that, as e.g. for drools, there is no such declaration needed in the top-level pom.xml [1]. The situation is similar there, one of the sub-modules excludes every java file from coverage statistics [2].

[1] https://github.com/kiegroup/drools/blob/master/pom.xml
[2] https://github.com/kiegroup/drools/blob/master/kie-dmn/kie-dmn-api/pom.xml#L68

Reply:
@rsynek 

Needed to analyze/upload code-coverage. Moved to scenario-simulation